src/universal/modules/teamDashboard/components/AgendaList/AgendaList.js

```diff
@@ -38,13 +38,33 @@ class AgendaList extends Component {
     visibleAgendaItemId: PropTypes.string,
     submittedCount: PropTypes.number,
     team: PropTypes.object.isRequired
-  };
+  }
 
   state = {
+    filteredAgendaItems: [],
     overflownAbove: false,
     overflownBelow: false
   };
 
+  componentWillMount() {
+    this.setFilteredAgendaItems(this.props);
+  }
+
+  componentWillReceiveProps(nextProps) {
+    const {team: {agendaItems, contentFilter}} = nextProps;
+    const {team: {agendaItems: oldAgendaItems, contentFilter: oldContentFilter}} = this.props;
+    if (agendaItems !== oldAgendaItems || contentFilter !== oldContentFilter) {
+      this.setFilteredAgendaItems(nextProps);
+    }
+  }
+
+  setFilteredAgendaItems = (props) => {
+    const {team: {agendaItems, contentFilter}} = props;
+    this.setState({
+      filteredAgendaItems: contentFilter ? agendaItems.filter(({content}) => content.match(contentFilter)) : agendaItems
```

Comment:
Down below you're matching against a regex constructed from the `contentFilter` string. Should we be doing that here, too?

Reply:
that's what `content.match` does. if it's not workin lemme know!